### DISCUSSION TITLE
error in installing atri framework

### DISCUSSION THREAD

**Discussion by Yashwanth9990** [Atri-Labs/atrilabs-engine#593] [Help: Installation & Start] [answered]
<img width="470" alt="Screenshot 2022-12-21 005422" src="https://user-images.githubusercontent.com/121105369/208749904-4f70b495-a1d8-4be2-bab5-073dfdcc38b6.png">

Installing atri...
Installing dependencies from Pipfile.lock (2599ed)...
To activate this project's virtualenv, run pipenv shell.
Alternatively, run a command inside the virtualenv with pipenv run.

what should i do now

**Comment by cruxcode** [comment]
It doesn't seem like you have any errors. Go ahead and follow the rest of the steps. Make sure you have `pipenv` installed and it's on the path environment variable.

**Comment by Yashwanth9990** [comment]
<img width="287" alt="Screenshot 2022-12-21 164247" src="https://user-images.githubusercontent.com/121105369/208892482-70daea31-f0ad-4635-a7bd-81751bbb3f47.png">
<img width="470" alt="Screenshot 2022-12-21 005422" src="https://user-images.githubusercontent.com/121105369/208892506-e1b7875f-cc0e-4d7d-af1b-410d1c6d5e14.png">

<img width="468" alt="Screenshot 2022-12-21 164532" src="https://user-images.githubusercontent.com/121105369/208892848-22a07ec8-01df-433b-ba97-5d13f84e8f3d.png">

in environmental path has was already formed , but the frame work was not installing

**Comment by darshitac11** [comment, marked as answer]
@Yashwanth9990 - Please follow the instructions shared [here](https://github.com/Atri-Labs/atrilabs-engine/discussions/338)

**Reply by Yashwanth9990** [reply]
Mam i have already followed the instructions but i can't get a solution please help me

**Reply by darshitac11** [reply]
Okay, let's start with prerequisites first: are you using Python version >=3.8 & <=3.10.8? Seems like you might be using Python 3.11

**Reply by Yashwanth9990** [reply]
yes mam i am using python 3.11

**Reply by darshitac11** [reply]
There are several problems with Python 3.11. This is why we recommend Python version >=3.8 & <=3.10.8

Please install Python version accordingly.

**Reply by Yashwanth9990** [reply]
Ok mam i will install 3.10 and i will follow the instructions and if the problem occurs i will revert back to you

**Reply by darshitac11** [reply]
Great, thanks!

**Reply by Yashwanth9990** [reply]
<img width="584" alt="Screenshot 2022-12-21 233313" src="https://user-images.githubusercontent.com/121105369/208973615-9756bd3f-c803-4be2-883c-397cc168d413.png">
<img width="282" alt="Screenshot 2022-12-21 233337" src="https://user-images.githubusercontent.com/121105369/208973630-b4fa2398-d49b-4779-acd5-45e0ab212161.png">

Now this is happening mam

**Reply by Yashwanth9990** [reply]
@darshitac11 madam any solution for this problem
